service.yml

```diff
@@ -6,8 +6,7 @@ codeowners:
 semaphore:
   enable: true
   pipeline_type: cp
-  downstream_projects: ["schema-registry", "metadata-service", "kafka-rest",
-    "confluent-security-plugins", "ce-kafka-http-server", "secret-registry",
-    "confluent-cloud-plugins"]
+  downstream_projects: ["schema-registry", "metadata-service",
```

Comment:
@sriman-confluent I think we would still need to keep kafka-rest as a downstream project here, instread we should remove `confluent-cloud-plugins` from kafka-rest downstream repo.